### PR TITLE
@dzucconi -> add artwork sorts

### PR DESCRIPTION
**Pull request by 1aurabrown**

### CODE REVIEW

File: schema/sorts/artwork_sorts.js [hunk 1]
```diff
@@ -28,6 +34,12 @@ export default {
       published_at_desc: {
         value: '-published_at',
       },
+      partner_updated_at_desc: {
+        value: '-partner_updated_at',
+      },
+      availability: {
+        value: '-availability',
```
Comment (dzucconi) [inline schema/sorts/artwork_sorts.js]: should have a `_desc` for consistency

**Comment by dzucconi** [comment]
I'd just add that `_desc` & merge

**Comment by dzucconi** [comment]
bump @1aurabrown

**Comment by 1aurabrown** [comment]
@dzucconi i missed this

**Pull request merged**